docs/.vitepress/theme/custom.css

```
@@ -112,6 +142,78 @@ kbd {
+  border-radius: 10px;
+  border: 1px solid transparent;
+  background: transparent !important;
+  color: var(--vp-c-text-1);
+  box-shadow: none !important;
+  overflow: visible;
+}
+
+.VPNavBarMenu .VPNavBarMenuLink:hover,
+.VPNavBarMenu .VPFlyout .button:hover {
+  border-color: transparent;
+  background: rgba(37, 99, 235, 0.08) !important;
```

Stylelint (17.2.0)

[error] Expected "0.08" to be "8%" (alpha-value-notation)

(alpha-value-notation)